lib/layers/presentation/settings_page/settings_page.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:wot_statistic/layers/presentation/settings_page/widgets/language_picker.dart';
+
+import '../../../common/theme/text_styles.dart';
+import 'bloc/settings_cubit.dart';
+
+class SettingsPage extends StatefulWidget {
+  const SettingsPage({Key? key}) : super(key: key);
+  static const String id = 'Settings';
+
+  @override
+  State<SettingsPage> createState() => _SettingsPageState();
+}
+
+class _SettingsPageState extends State<SettingsPage> {
+  @override
+  Widget build(BuildContext context) {
+    bool _switchState = context.read<SettingsCubit>().isDarkTheme;
+    SettingsCubit themeCubit = context.read<SettingsCubit>();
+    SettingsState state = themeCubit.state;
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          SettingsPage.id,
+          style: appBarTitle(context),
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            const SizedBox(height: 25),
+            Center(
+                child: Text(
+              "Theme Settings",
+              style: onPrimarySubtitle(context),
+            )),
+            Divider(
+                thickness: 3, color: Theme.of(context).colorScheme.secondary),
+            Row(
```

Comment:
I would recommend extracting such large parts of UI in separate widgets or private methods to simplify the reading.